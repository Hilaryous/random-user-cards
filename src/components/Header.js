// @flow
import styled from 'react-emotion'

const Header = styled('h1')({
  color: '#294762',
  opacity: '0.8',
  lineHeight: 1.5,
  fontSize: 20,
  textTransform: 'capitalize',
})

export default Header
