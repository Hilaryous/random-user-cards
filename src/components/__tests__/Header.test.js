/* eslint-disable function-paren-newline */
import React from 'react'
import renderer from 'react-test-renderer'
import Header from '../Header'

describe('Header', () => {
  it('shows the correct snapshot tree for the Header component', () => {
    const tree = renderer.create(
      <Header />,
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
