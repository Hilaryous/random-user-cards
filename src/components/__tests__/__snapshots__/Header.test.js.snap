// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header shows the correct snapshot tree for the Header component 1`] = `
<h1
  className="css-1ce92xy"
/>
`;
